src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+class FigureSupplier {
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the empty line at the start of the class definition.